picard/track.py

```diff
@@ -64,6 +66,52 @@
 }
 
 
+class TagGenreFilter:
+
+    def __init__(self, setting=None):
```

Comment:
I would suggest to change this to not pass the settings but instead `setting['genres_filter']`. This way this class could be easily re-used by other genre plugins with own filter list (or still the default one if preferred depending on the plugin).

Reply:
Done in 1aee66ccff044f